---
title: "README: Examining the expression of skin targets at the single cell level"
author: 
- Andrew Davis, PhD, Unilever Human Biology
date: "`r format(Sys.time(), '%d %B %Y')`"
output:
  rmarkdown::html_document:
    toc: TRUE
---

# Overview

This repository contains an exploratory analysis of the single cell RNA-seq data generated by [Sol√© Boldo et al.](https://www.nature.com/articles/s42003-020-0922-4).


# Metadata:


# Input Data

I downloaded the Seurat file from GEO for GSE130973: [GSE130973_seurat_analysis_lyko.RData.gz](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE130973&format=file&file=GSE130973%5Fseurat%5Fanalysis%5Flyko%2ERData%2Egz)

Note that, according to the authors, this was an early version analysis that was since updated when they published. Therefore I needed to re-analyze this data using the integration workflow described by the Seurat team here: [https://satijalab.org/seurat/v3.0/integration.html](https://satijalab.org/seurat/v3.0/integration.html).

# Analysis pipeline:



# Output:

The remaining output files were saved to the "Results" folder in this repository.

```{r, echo=FALSE}
list.files(path="Results")
```

# Mechanics

The analysis in this project proceeds by first cleaning the Results directory using 95_Make_Clean.Rmd. Then, one should run all of the code chunks in 99_Run_All.Rmd (this runs the each of the Rmd scripts in numerical sequence). The IPA analyses were run within IPA.

The Rmd scripts are as follows:

1. 1.03_Gather-and-Tidy-Data.Rmd

# Summary:

## Assigning cell type to clusters (v.1.0)
- Comparing my AD20 clusters to Sole-Boldo and Tabib. Also used CellMarker database as reference to manually annotate clusters. Lower confidence labels - 5_Secretory-reticular fibroblast, 14_Lymphocytes, 17_Endothelial,19_Secretory_Epith

0: HLA-DRB1, HLA-DPB1, LYZ, HLA-DQA1, HLA-DRA - 0_Macrophages/DC
1: SELE, TMFSF1, DARC, IFI27, SPARCL1, TSC22D1 - 1_Vascular EC
2: APCDD1, LEPR, WIF1, COL6A1, PTGDS - 2_Secretory-reticular fibroblasts
3: KRTDAP	KRT1	DMKN	KRT10	LGALS7B	- 3_Differentiated Keratinocyte
4: SLPI	IGFBP6	CFD	PLA2G2A	WISP2 - 4_Secretory-Papillary fibroblasts
5: HSPA1B	SERPINE2	PPAP2B	CXCL12	- 5_Secretory-reticular fibroblast
6: CD52	IL32	CD69	CD3D	CCL5 - 6_T Cells
7: KRT14	S100A2	KRT5	SERPINB2	AREG	- 7_Keratinocytes undiff.
8: APOE	CXCL12	CXCL2	CXCL1	CCL19 -	8_Pro-inflammatory fibroblasts
9: TAGLN	ACTA2	TPM2	MYL9	RGS5	- 9_Pericyte
10: ASPN	COL1A1	POSTN	COL3A1	APOD -	10_Mesenchymal fibroblasts
11: HLA-DRB5	HLA-DQA2	HLA-DQB2	IL8	HLA-DRA	- 11_Macrophage/DC
12: WISP2	FBLN1	DCN	COL1A2	COL1A1 - 12_Fibroblast
13: CCL21	TFF3	CLDN5	MMRN1	FABP4	- 13_Lymphatic EC
14: CD52	CXCR4	IL32	IGLL5	IGJ	- 14_Lymphocytes
15: KIAA0101	HIST1H4C	STMN1	HMGB2	HMGN2	- 15_Keratinocytes undiff.
16: ACTA2	TAGLN	TPM2	ACTG2	MYLK - 16_Smooth Muscle
17: RGS5	NDUFA4L2	SPRY1	TM4SF1	TSC22D1 - 17_Endothelial
18: DCT	TYRP1	MLANA	PMEL	CAPN3	- 18_Melanocytes
19: S100A1	C2orf82	NDRG2	CHCHD10	GAPDH - 19_Secretory_Epith.

#### Nothing matches to neural cell or corneocyte?

## Re-label clusters
AD.cluster.ids<- c("0_Macrophages/DC","1_Vascular EC","2_Secretory-reticular fibroblasts","3_Differentiated Keratinocyte-1","4_Secretory-Papillary fibroblasts","5_Secretory-reticular fibroblast","6_T Cells","7_Keratinocytes undiff.","8_Pro-inflammatory fibroblasts","9_Pericyte","10_Mesenchymal fibroblasts","11_Macrophage/DC","12_Fibroblast","13_Lymphatic EC","14_Lymphocytes","15_Keratinocytes undiff.","16_Smooth Muscle","17_Endothelial","18_Melanocytes","19_Secretory_Epith.")

#### Need to double check: 


## Notes/To do:


1. I noticed multiple mt-encoded and ribosomal genes in the old vs. young comparisons, particularly for fibroblasts. These genes appear to be lower in the older samples? Is this real or reflective of technical issue that correlates in the age of the subjects? Should (can) I regress the % mtDNA and/or ribosomal genes during normalization?
2. How can I summarize the % of cell types between young and old? Like Sole-Boldo Fig 4A
3. Is COL17A1 or SOD3+ cells proportion/levels diferent in the young vs old subjects?
4. Run IPA Core Analysis on the young vs. old intracellular comparisons
5. How do the Sole-Boldo clusters compare to Tabib? How do I integrate the 2 studies?
6. Re-analyze Tabib using Seurat only - are the clusters still very different?
7. GO analysis of the COL17A1 and SOD3 clusters
8. Make ppt summarizing results
9. Shiny app?

### Session information

```{r session_info}
sessionInfo()
```

This document was processed on: `r Sys.Date()`.





